eventservice/api/controllers/events.py

```diff
@@ -41,10 +43,28 @@
 
 
 def search():
+    group_ids = list()
+    include_private_events = False
+    # user UserAuth request
+    if 'user_token_data' in g:
+        include_private_events = True
+        auth_resp = g.user_token_data
+        uin = auth_resp.get('uiucedu_uin')
+        url = "%s%s/groups" % (cfg.GROUPS_BUILDING_BLOCK_ENDPOINT, uin)
+        headers = {"Content-Type": "application/json",
+                  "ROKWIRE_GS_API_KEY": cfg.ROKWIRE_GROUPS_API_KEY}
+
+        req = requests.get(url, headers=headers)
+        if req.status_code == 200:
```

Comment:
If there is an error communicating with Groups Building Block, Events Building Block should return HTTP 500 with an appropriate message in the response body.